### PR TITLE
Port to Python

### PR DESCRIPTION
**Pull request by broman**

### CODE REVIEW

**Comment by thecatitude** [comment]
what the hell is this crap

**Comment by thecatitude** [comment]
Rejected. Terrible API design, this language should not be used for anything. Incorrect formatting. Licensing updated without permissions from the original license holder. Please delete and rethink major life decisions. Replaced API code doesn't mimic the original, and lacks features present in the original. -10/10

~ Mathematica

**Comment by ryanbroman** [comment]
Please keep contributions on Github professional, thecatitude. Thank you!

**Comment by ryanbroman** [comment]
Awaiting a reply.

**Comment by broman** [comment]
Bloat

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+name = "isidiot"
```
Comment (R0r71z) [inline __init__.py]: Add extra line here.

File: isidiot.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def is_idiot(thing):
+    return True
```
Comment (R0r71z) [inline isidiot.py]: Same here.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    description="Determines if somebody is an idiot",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/gabixdev/is-idiot",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+    ],
+)
```
Comment (R0r71z) [inline setup.py]: Same here.

Comment (broman) [reply setup.py]: No

File: __init__.py (hunk 1 above)
Comment (broman) [reply __init__.py]: No

**Comment by broman** [comment]
Housekeeping! Cleaning old PR's

**Pull request closed without merging**